Code/GraphMol/ChemReactions/ReactionRunner.cpp

```diff
@@ -708,8 +703,8 @@ void generateProductConformers(Conformer *productConf, const ROMol &reactant,
                                  "atom, coordinates need to be revised\n";
     }
     // is this reliable when multiple product atom mapping occurs????
-    for (unsigned i = 0; i < prodIdxs.size(); i++) {
-      productConf->setAtomPos(prodIdxs[i], reactConf.getAtomPos(pr->first));
+    for (unsigned int prodIdx : prodIdxs) {
+      productConf->setAtomPos(prodIdx, reactConf.getAtomPos(pr->first));
     }
```

Comment:
I don't think it would.  I is typically an index, using this style of loop it is equivalent to 

```
prodIdxs[i]
```

which nicely was converted to prodIdx!